amqp_test.go

```diff
@@ -214,13 +211,13 @@ func TestSendStopRequest(t *testing.T) {
 	coord := make(chan int)
 	invID := "test"
 	handler := func(d amqp.Delivery) {
-		d.Ack(false)
+		_ = d.Ack(false)
 		actual = d.Body
 		coord <- 1
 	}
 	key := StopRequestKey(invID)
 	client.AddConsumer(exchange(), exchangeType(), "test_queue4", key, handler, 0)
-	client.SendStopRequest(invID, "test_user", "this is a test")
+	_ = client.SendStopRequest(invID, "test_user", "this is a test")
```

Comment:
I'm probably missing something, but what's the reason behind these small refactors?

Reply:
Ah, that was just a small change to fix a linter warning. 🙂